server/views/admin-add-article.hbs

```diff
@@ -1,24 +1,20 @@
 <form method="post" >
     <div class="form-group has-success">
-        <h5 class="text-success" >Title:</h5>
-        <input  class="form-control" id="inputSuccess1" name="title"/>
-    </div>
-    <div class="form-group has-success">
-        <h4> Categories:</h4>
-        <select name="category" class="form-control">
+        <h5 class="text-success mt-5" >Title:</h5>
```

Comment:
Why did you remove form-groups? Each form element is wrapped in a form-group in bootstrap - look here http://getbootstrap.com/docs/4.0/components/forms/